src/ctx.cpp

```diff
@@ -1364,11 +1364,12 @@ llvm::Value *FunctionEmitContext::MasksAllEqual(llvm::Value *v1, llvm::Value *v2
 #else
     if (g->target->getArch() == Arch::wasm32) {
         llvm::Function *fmm = m->module->getFunction("__wasm_cmp_msk_eq");
-        return CallInst(fmm, NULL, {v1, v2}, LLVMGetName("wasm_cmp_msk_eq", v1, v2));
+        return CallInst(fmm, NULL, {v1, v2}, (llvm::Twine("wasm_cmp_msk_eq", v1->getName()) + v2->getName()));
```

Comment:
Should we have underscores inserted between "wasm_cmp_msk_eq", v1->getName() and v2->getName()?

Reply:
I changed the way I added strings and added the underscore